package.json

```diff
@@ -27,23 +27,24 @@
   "author": "GitHub, Inc",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@xmldom/xmldom": "^0.8.7",
-    "jsonpath-plus": "^7.2.0",
-    "xpath": "^0.0.32",
+    "@actions/core": "^1.10.1",
+    "@xmldom/xmldom": "^0.8.10",
```

Comment:
`npm install -S @types/xmldom`

Reply:
xmldom includes type information in the package